resources/testharness.js

```diff
@@ -1436,11 +1436,28 @@ policies and contribution forms [3].
                 NotAllowedError: 0
             };
 
-            if (!(name in name_code_map)) {
-                throw new AssertionError('Test bug: unrecognized DOMException code "' + code + '" passed to assert_throws()');
+            var code_name_map = {};
+            for (let [k, v] of Object.entries(name_code_map)) {
```

Comment:
Let's keep testharness.js working in more older browsers if we can and just rely on ES5 syntax.

Reply:
Done. If that's the long-term intention (stay es5), might be good to get a lint set up.

Reply:
[And also documented.](https://github.com/web-platform-tests/wpt/pull/6333)